cernopendata/modules/fixtures/data/records/cms-SW-FIXMEfilename.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "abstract": {
+      "description": "This is a CMSSW module for producing flat tuples containing jet properties from 13 TeV Run2 MC samples. The code is intended to run inside the CMS Open Data environment."
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Kallonen, Kimmo",
+        "orcid": "0000-0001-9769-7163"
+      }
+    ],
+    "collections": [
+      "CMS-Validation-Utilities"
+    ],
+    "date_created": [
+      "2019"
+    ],
+    "date_published": "2019",
+    "distribution": {
+      "formats": [
+        "gz"
+      ],
+      "number_files": 1,
+      "size": 11111111
+    },
+    "doi": "FIXME",
```

Comment:
We can only mint DOI once the example will be having some GitHub tag.  Note that they may move to a different GitHub organisation... so I'll merge without DOI parts for now.

Reply:
Amended